### ISSUE TITLE
Codestyle: unnecessary conversion affine -> projective -> affine

### ISSUE THREAD

**Issue by webmaster128** [thibmeu/drand-rs#5]
I think the code https://github.com/thibmeu/drand-rs/blob/a2fd6493ed2f01755a10cf9657faeb91fd2fcbde/drand_core/src/bls_signatures.rs#L48-L53

contains an unnecessary circle G2Affine -> G2Projective -> G2Affine. It should be possible to just do

```rust
// type G2Affine
let hash_on_curve = mapper.hash(hash).map_err(|_| anyhow!("hash cannot be mapped to G1"))?;
```

**Comment by thibmeu** [comment]
Thanks for the report.
Fixed in 264db2322c7d7ed268aaab6cfeea2a0d235cf2a5

**Comment by webmaster128** [comment]
Sweet!
